CHANGES.rst

```diff
@@ -6,6 +6,8 @@ astropy-helpers Changelog
 
 - Removing deprecated test_helpers.py file. [#369]
 
+- Removing ez_setup.py file and require setuptools 1.0 or later. [#384]
```

Comment:
(nitpick) to be grammatically correct I think this needs to be "requiring"?  Or Removing->remove.